WalletWasabi.Fluent/Views/Wallets/Settings/WalletCoinJoinSettingsView.axaml

```diff
@@ -10,46 +10,45 @@
              x:Class="WalletWasabi.Fluent.Views.Wallets.Settings.WalletCoinJoinSettingsView"
              x:CompileBindings="True">
 
-  <StackPanel Spacing="20">
+  <StackPanel Classes="settingsLayout">
 
-    <StackPanel Classes="settingsLayout">
+    <DockPanel>
+      <TextBlock Text="Automatically start coinjoin" />
```

Comment:
I would rename this to be clear this setting is related to the `threshold` one.

```suggestion
      <TextBlock Text="Auto-start coinjoin" />
```

Reply:
It seems like they are not related: https://github.com/zkSNACKs/WalletWasabi/pull/13067#issuecomment-2126356186